docs/excel/excel-add-ins-workbooks.md

```diff
@@ -0,0 +1,170 @@
+		}
+	});
+}).catch(errorHandlerFunction);
+```
+
+The `protect` method also has an optional string parameter, `password`. This string represents the password needed for a user to bypass protection and change the workbook's structure.
+
+Protection can also be set at the worksheet level. Protection on the worksheet guards against unwanted data editing. For more information, the Data protection section of the [Work with Worksheets using the Excel JavaScript API](excel-add-ins-worksheets.md#data-protection) article.
+
+The article [Protect a workbook](https://support.office.com/article/Protect-a-workbook-7E365A4D-3E89-4616-84CA-1931257C1517) has more information about workbook protection and how to change it through the Excel UI.
+
+## Document properties
+
+A workbook has access to the Office file metadata, which is known as the [document properties](https://support.office.com/article/View-or-change-the-properties-for-an-Office-file-21D604C2-481E-4379-8E54-1DD4622C6B75). The workbook's `properties` property is a [DocumentProperties](/javascript/api/excel/excel.documentproperties) object containing these metadata values. The following example shows how to set the author property.
+
+```js
+Excel.run(function (context) {
+	var docProperties = context.workbook.properties;
+	docProperties.author = "Alex";
+	return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+You can also define custom properties. The `DocumentProperties` object has `custom`: a map defining key-value pairs for user-defined properties. The following example shows how to create a custom "Introduction" property with the value "Hello", then retrieve it.
+
+```js
+Excel.run(function (context) {
+	var customDocProperties = context.workbook.properties.custom;
+	customDocProperties.add("Introduction", "Hello");
+	return context.sync();
+}).catch(errorHandlerFunction);
+
+[...]
+
+Excel.run(function (context) {
+	var customDocProperties = context.workbook.properties.custom;
+	var customProperty = customDocProperties.getItem("Introduction");
+	customProperty.load("key, value");
+
+	return context.sync().then(function() {
+        console.log("Custom key  : " + customProperty.key); // "Introduction"
+        console.log("Custom value : " + customProperty.value); // "Hello"
+	});
+}).catch(errorHandlerFunction);
+```
+
+## Document settings
+
+A workbook's settings are similar to the collection of custom properties. The difference is settings are unique to a single Excel file and add-in pairing, whereas properties are solely connected to the file. The following example shows how to create and access a setting.
+
+```js
+Excel.run(function (context) {
+	var settings = context.workbook.settings;
+	settings.add("NeedsReview", true);
+	var needsReview = settings.getItem("NeedsReview");
+	needsReview.load("value");
+
+	return context.sync().then(function() {
+		console.log("Workbook needs review : " + needsReview.value);
+	});
+}).catch(errorHandlerFunction);
+```
+
+## Calculations
```

Comment:
Consider adding H3 headings to separate the three topics covered in this section:  
- Set calculation mode
- Set calculation type
- Temporarily suspend calculations